foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/QueryResponse.java

```diff
@@ -28,12 +28,18 @@
  */
 public class QueryResponse implements ActionResponse {
     private List<Document> documents;
+    private long totalHits;
 
     public QueryResponse() {
     }
 
     public QueryResponse(List<Document> documents) {
```

Comment:
AAh .. the method .. my bad, I thoguht you were talking about totalHits
attribute.

----
Upset the established order, and everything becomes chaos. I'm an agent of
chaos. Oh and you know the thing about chaos, it's fair.

On Wed, Dec 9, 2015 at 12:46 PM, Rishabh <notifications@github.com> wrote:

> In
> foxtrot-common/src/main/java/com/flipkart/foxtrot/common/query/QueryResponse.java
> <https://github.com/Flipkart/foxtrot/pull/99#discussion_r47057558>:
>
> > @@ -28,12 +28,18 @@
> >   */
> >  public class QueryResponse implements ActionResponse {
> >      private List<Document> documents;
> > +    private long totalHits;
> >
> >      public QueryResponse() {
> >      }
> >
> >      public QueryResponse(List<Document> documents) {
>
> @santanusinha <https://github.com/santanusinha> I am not asking for
> moving it to action response. I am asking to completely remove this method
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Flipkart/foxtrot/pull/99/files#r47057558>.
>